dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2895,9 +2881,9 @@ def test_filter_nulls(tmpdir, filters, op, length, split_row_groups, engine):
 @PYARROW_MARK
 @pytest.mark.parametrize("split_row_groups", [True, False])
 def test_filter_isna(tmpdir, split_row_groups):
-    if pa_version < parse_version("6.0.0"):
-        pytest.skip("PyArrow>=6.0.0 needed for null filtering")
-
+    if engine == "pyarrow" and parse_version(pa.__version__) < parse_version("8.0.0"):
```

Comment:
```suggestion
    if parse_version(pa.__version__) < parse_version("8.0.0"):
```
Oops - My fault. No `engine` param in this test (only testing pyarrow).

Reply:
Should I add `engine=` or are these only supported for the `pyarrow` engine?

Reply:
I'd leave out the engine param for now. These tests assume row-wise filtering, which I believe is possible with the fastparquet engine, but is not yet exposed/active. I can revisit this separately since that question is out of scope for this PR.